### ISSUE TITLE
fix_in_frame_stop_codon_genes.py ERROR

### ISSUE THREAD

**Issue by francicco** [Gaius-Augustus/BRAKER#323]
Hi,

Buring a BRAKER run I get this error at the `fix_in_frame_stop_codon_genes.py` step:

`Error in file /work/tk19812/software/augustus-3.3.3/scripts/fix_in_frame_stop_codon_genes.py at line 243: Return code of subprocess was 1['/bin/grep', 'Scf0000925', '/work/tk19812/TetramoriumProject/Talp_GenomeReassemblies/wp96bd/wp96bd.BRAKER_prediction/hintsfile.gff.Ppri5']`

I don't quite understand how I can fix this.
Any help?

Cheers
F

**Comment by tomasbruna** [comment]
Hi @francicco,

I think this problem was solved before in AUGUSTUS (see https://github.com/Gaius-Augustus/BRAKER/issues/264), can you make sure you are working with the latest version of `fix_in_frame_stop_codon_genes.py`? You can download it from here https://github.com/Gaius-Augustus/Augustus/blob/master/scripts/fix_in_frame_stop_codon_genes.py.

Best,
Tomas

**Comment by francicco** [comment]
Yes, I think I fix it. Now I have another problem, but I'm gonna open a new ticket
Thanks a lot
F
